modules/engage-theodul-plugin-tab-transcript/src/main/resources/static/templates/desktop.html

```diff
@@ -6,13 +6,11 @@
 	<div id="engage_transcript_tab">
 		<% if(_.size(vttObjects) > 0) { %>
 			<div id="search-bar">
-				<form>
-					<div class="controls">
-						<label for="transcript_tab_search"><%= search_str %>:</label>
-						<input type="text" id="transcript_tab_search" placeholder="<%= search_placeholder_str %>">
-						<input type="reset" value="Clear" id="clear_transcript_tab_search">
-					</div>
-				</form>
+				<div class="controls">
+					<label for="transcript_tab_search"><%= search_str %>:</label>
+					<input type="text" id="transcript_tab_search" placeholder="<%= search_placeholder_str %>">
+					<button onclick="document.getElementById('transcript_tab_search').value = ''">Clear</button>
```

Comment:
Please put this in a function
onclick="document.getElementById('transcript_tab_search').value = ''"